Integrations/ESPHome/AIR-1.yaml

```diff
@@ -103,6 +103,19 @@ number:
     update_interval: never
     step: 0.1
     mode: box
+  - platform: template
+    name: SEN55 Humidity Offset
+    id: sen55_humidity_offset
+    restore_value: true
+    initial_value: 0
+    min_value: -70.0
+    max_value: 70.0
+    entity_category: "CONFIG"
+    unit_of_measurement: "°C"
```

Comment:
@TrevorSchirmer Shouldn't this be "%" instead?

Reply:
Yes it should, thanks for catching that. It won't matter for the calculation but will display wrong in Home Assistant. I'll get it updated in the firmware.

Reply:
Thanks for pointing this out, I've corrected it in the latest firmware I just merged.